src/android/LocalNotification.java

```diff
@@ -90,6 +95,14 @@ public void initialize (CordovaInterface cordova, CordovaWebView webView) {
     public void onResume (boolean multitasking) {
         super.onResume(multitasking);
         deviceready();
+
+        Context context = cordova.getActivity().getApplicationContext();
```

Comment:
Should we not use a runtime request permission instead of navigating to the settings page? Permissions like for pictures and Geo-location at the start of the app?